src/mame/philips/mcd212.cpp

```diff
@@ -1434,12 +1406,10 @@ void mcd212_device::device_start()
 		m_delta_uv_lut[d] = s_dyuv_deltas[d >> 4];
 	}
 
-	for (uint16_t w = 0; w < 3 * 0xff; w++)
+	for (uint16_t w = 0; w < 0x300; w++)
 	{
-		const uint8_t limit = (w < 0xff + 16) ?  0 : w <= 16 + 2 * 0xff ? w - 0x10f : 0xff;
-		m_dyuv_limit_r_lut[w] = limit << 16;
-		m_dyuv_limit_g_lut[w] = limit << 8;
-		m_dyuv_limit_b_lut[w] = limit;
+		const uint8_t limit = (w < 0x100) ? 0 : w < 0x200 ? w - 0x100 : 0xff;
+		m_dyuv_limit_rgb_lut[w] = limit;
```

Comment:
Please put use parentheses around non-trivial arguments to ternary conditional operators to make them easier to parse mentally, like:
```c++
		const uint8_t limit = (w < 0x100) ? 0 : (w < 0x200) ? (w - 0x100) : 0xff;
```